app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Section/AdminMediaGalleryCatalogUiCategoryGridSection.xml

```diff
@@ -14,5 +14,6 @@
         <element name="image" type="text" selector="//tr//td[count(//div[@data-role='grid-wrapper']//tr//th[contains(., 'Image')]/preceding-sibling::th) +1]//img[contains(@src, '{{file}}')]" parameterized="true"/>
         <element name="columnValue" type="text" selector="//tr//td[count(//div[@data-role='grid-wrapper']//tr//th[contains(., '{{columnName}}')]/preceding-sibling::th) +1 ]//div" parameterized="true"/>
         <element name="edit" type="button" selector="//tr[td//text()[contains(., '{{categoryName}}')]]//td[count(//div[@data-role='grid-wrapper']//tr//th[contains(., 'Action')]/preceding-sibling::th) +1 ]//*[text()='{{actionButton}}']" parameterized="true"/>
+        <element name="noDataMessage" type="text" selector="div.no-data-message-container"/>
```

Comment:
This section is used for the category grid elements.

Can you please move this element to new `app/code/Magento/MediaGalleryUi/Test/Mftf/Section/AdminMediaGalleryGridSection.xml`.

It would be good also to delete the `AdminEnhancedMediaGalleryGridImagePositionSection` and move the `nthImageInGrid` to new `AdminMediaGalleryGridSection`

Reply:
moved to new section AdminMediaGalleryGridSection